File: PHP Excel Examples/WorksheetsApiDemo/Protect/unProtectWorksheet.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;
use Spire\Cloud\Excel\Sdk\Model\ProtectSheetParameter;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "UnprotectWorksheet.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet1";
$protectParameter = new ProtectSheetParameter();
$protectParameter->setPassword("123");
$protectParameter->setAllowInsertingRow("false");
$result = $apiInstance->unprotectWorksheet($name, $sheetName,$protectParameter,$folder,$storage);

File: Python Excel Examples/WorksheetsApiDemo/protect/unProtectWorksheet.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "ProtectWorksheet.xlsx"
storage = ""
folder = "ExcelDocument"
sheetName = "Sheet1"
protectParameter = spirecloudexcel.models.ProtectSheetParameter()
protectParameter.password = "123"
protectParameter.allow_inserting_row = "false"
api.unprotect_worksheet(name, sheetName, protectParameter, storage=storage, folder=folder)